controllers/grid/users/reviewer/ReviewerGridRow.php

```diff
@@ -182,11 +184,19 @@ public function initialize($request, $template = null)
                 )
             );
 
-            $user = $request->getUser();
+            // Simplified "Login as" action check
+            $currentUser = $request->getUser();
+            $reviewerId = $reviewAssignment->getReviewerId();
+
+            // Get the context (journal) to check for the Manager role
+            $context = $request->getContext();
+            $contextId = $context ? $context->getId() : 0;
```

Comment:
instead of `0`, we should use `Application::SITE_CONTEXT_ID` that point to a site as there can be no context with ID `0` .